spec/routes/api/cli/golden-files/help -r.txt

```diff
@@ -77,6 +82,57 @@ Options:
     -u, --usage                      only show usage
 
 
+Usage: ubi lb subcommand [...]
+
+Subcommands: list
+
+
+Usage: ubi lb location/(lb-name|_lb-ubid) subcommand [...]
+
+Subcommands: attach-vm create destroy detach-vm show update
```

Comment:
Similar here, subcommands should have an order (looks like alpha again).

Reply:
The alpha order here is enforced.